tasmota/tasmota_xdrv_driver/xdrv_90_esp32_dingtian_relay.ino

```diff
@@ -83,31 +91,31 @@ uint32_t DingtianReadWrite(uint32_t outputs)
  * Driver initialisation
  */
 
-#define DINGTIAN_SET_OUTPUT(pin,value)  { pinMode((pin), OUTPUT); digitalWrite((pin), (value)); }
-#define DINGTIAN_SET_INPUT(pin)         { pinMode((pin), INPUT); }
-
 void DingtianInit(void) {
   if (PinUsed(GPIO_DINGTIAN_CLK, GPIO_ANY) && PinUsed(GPIO_DINGTIAN_SDI) && PinUsed(GPIO_DINGTIAN_Q7)
-   && PinUsed(GPIO_DINGTIAN_PL) && PinUsed(GPIO_DINGTIAN_RCK)) {
+   && (PinUsed(GPIO_DINGTIAN_PL) || PinUsed(GPIO_DINGTIAN_OE)) && PinUsed(GPIO_DINGTIAN_RCK)) {
```

Comment:
I don't get that
By default PL should always been set for all models of board
OE should be an optional pin only for new boards

Reply:
PL is driven via and inverter connected to RCLK.